src/Hl7.Fhir.Base/Model/Generated/Signature.cs

```diff
@@ -103,14 +103,14 @@ public Hl7.Fhir.Model.Instant WhenElement
     /// <summary>
     /// Who signed. Note: Since R5 the cardinality is expanded to 0..1 (previous it was 1..1)
     /// </summary>
-    [FhirElement("who", InSummary = true, Order = 50, Choice = ChoiceType.DatatypeChoice)]
-    [FhirElement("who", InSummary = true, Order = 50, Since = FhirRelease.R4)]
-    [CLSCompliant(false)]
-    [References("Practitioner", "RelatedPerson", "Patient", "Device", "Organization")]
-    [References("Practitioner", "PractitionerRole", "RelatedPerson", "Patient", "Device", "Organization",Since=FhirRelease.R4)]
+    [FhirElement("who", InSummary=true, Order=50, Choice = ChoiceType.DatatypeChoice)]
```

Comment:
Should the doccomment mention the change of choice -> fixed type?

Reply:
I changed it now like this:
```
   /// <summary>
    /// Who signed.
    /// Note 1: Since R4 the type of this element should be a fixed type (ResourceReference). For backwards compatibility it remains of type DataType.
    /// Note 2: Since R5 the cardinality is expanded to 0..1 (previous it was 1..1).
    /// </summary>
 ````